ratelimiter/build.gradle

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License.
+ *
+ */
+
+plugins {
+    id 'net.researchgate.release' version '2.8.0'
+}
+
+apply from: "$rootDir/../common-gradle-scripts/docker.gradle"
+
+release {
+    tagTemplate = 'ratelimiter-$version'
+
+    git {
+        requireBranch= "main"
+        pushToRemote= "origin"
+    }
+}
+
+allprojects {
+    group = project.group
+    version = project.version
+}
+
+task build{
+    dependsOn docker_build
+}
+
+afterReleaseBuild.dependsOn "docker_push"
```

Comment:
might need to add a GitHub workflow related changes to build rate limiter and also add to build-apk.sh?

Reply:
sure. I'll add this in my next pr. Thanks for pointing it out.